resources/wikia/modules/domCalculator.js

```diff
@@ -14,7 +14,7 @@ define('wikia.domCalculator', [
 			element = element.offsetParent;
 		} while (element !== null);
 
-		if (elementWindow.frameElement) {
+		if (elementWindow && elementWindow.frameElement) {
```

Comment:
how about removing `element.ownerDocument.parentWindow`? It's not supported by any browser we support.